pkg/configui/configui.go

```diff
@@ -59,15 +62,18 @@ func BuildMainMenu(cfg *config.Instance, pages *tview.Pages, app *tview.Applicat
 		AddItem("Scan mode", "Set scanning options", '4', func() {
 			pages.SwitchToPage("scan")
 		}).
-		AddItem("Systems", "Not implemented yet", '5', func() {
+		AddItem("Manage tags", "Read and write nfc tags", '5', func() {
+			pages.SwitchToPage("tags")
 		}).
-		AddItem("Launchers", "Not implemented yet", '6', func() {
+		AddItem("Systems", "Not implemented yet", '6', func() {
 		}).
-		AddItem("ZapScript", "Not implemented yet", '7', func() {
+		AddItem("Launchers", "Not implemented yet", '7', func() {
 		}).
-		AddItem("Service", "Not implemented yet", '8', func() {
+		AddItem("ZapScript", "Not implemented yet", '8', func() {
 		}).
-		AddItem("Mappings", "Not implemented yet", '9', func() {
+		AddItem("Service", "Not implemented yet", '9', func() {
+		}).
+		AddItem("Mappings", "Not implemented yet", '0', func() {
```

Comment:
Menu needs to be reorganized, i built something from the types of the configuration, but i m sure we can find a nicer menu with less items, as soon as we all features